localization.md

```diff
@@ -1,22 +1,33 @@
+
+Turn instructions are primarily designed to be announced by either the Mapbox Voice API (powered by [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/SupportedLanguage.html)) or [TextToSpeech](https://developer.android.com/reference/android/speech/tts/TextToSpeech). By default, this SDK uses the Mapbox Voice API, which requires an Internet connection at various points along the route. If the Voice API lacks support for the turn instruction language or there is no Internet connection, TextToSpeech announces the instructions instead. 
+
+By default, distances are given in the predominant measurement system of the system region, which may not necessarily be the same region in which the user is traveling. To override the measurement system used in spoken instructions, set the `MapboxNavigationOptions.Builder#unitType` property when calculating the route with which to start navigation.
+
+The upcoming road or ramp destination is named according to the local or national language. In some regions, the name may be given in multiple languages.
+
 ## Supported languages
 
-| Language   | User interface | Mapbox Voice API | Android TextToSpeech
-|------------|:--------------:|:----------------:|:---------------------:
-| Catalan    | -              | —                | —
-| Chinese    | -              | ✅               | ✅
-| Danish     | -              | ✅               | ✅
-| Dutch      | -              | ✅               | ✅
-| English    | ✅             | ✅               | ✅
-| French     | -              | ✅               | ✅
-| German     | -              | ✅               | ✅
-| Hebrew     | -              | -                | —
-| Hungarian  | -              | —                | ✅
-| Italian    | -              | ✅               | ✅
-| Portuguese | -              | ✅               | ✅
-| Polish     | —              | ✅               | ✅
-| Romanian   | —              | ✅               | ✅          
-| Russian    | -              | ✅               | ✅
-| Spanish    | -              | ✅               | ✅
-| Swedish    | -              | ✅               | ✅
-| Turkish    | —              | ✅               | ✅
-| Vietnamese | -              | ✅               | ✅
+The table below lists the languages that are supported for user interface elements and for spoken instructions.
+
+| Language   | User interface | [Spoken instructions][osrmti] | Remarks
+|------------|:--------------:|:-----------------------------:|--------
+| Bengali    | ✅             | — 
+| Czech      | ✅             | -
+| Danish     | ✅             | ✅
+| French     | ✅             | ✅
+| English    | ✅             | ✅
+| French     | ✅             | ✅
+| German     | ✅             | ✅
+| Hebrew     | ✅             | - 
+| Italian    | ✅             | ✅
+| Korean     | ✅             | —
+| Portuguese(Portugese) | ✅  | ✅
```

Comment:
“Portugese” is misspelled. We support both Brazilian and European Portuguese in the iOS navigation SDK, but I decided not to list the dialects in [the iOS language support document](https://github.com/mapbox/mapbox-navigation-ios/blob/master/docs/guides/Localization%20and%20Internationalization.md), because users of both dialects will still get Portuguese instead of English.